tools/importer/import-support.js

```diff
@@ -0,0 +1,154 @@
+      title.remove();
+    }
+
+    const headingTwos = main.querySelectorAll('span.cmp-text__heading-h2');
+    if (headingTwos.length > 0) {
+      headingTwos.forEach((h2) => {
+        const newH2 = document.createElement('H2');
+        newH2.innerHTML += h2.innerHTML;
+        h2.replaceWith(newH2);
+      });
+    }
+
+    const headingThrees = main.querySelectorAll('span.cmp-text__heading-h3');
+    if (headingThrees.length > 0) {
+      headingThrees.forEach((h3) => {
+        const newH3 = document.createElement('H3');
+        newH3.innerHTML += h3.innerHTML;
+        h3.replaceWith(newH3);
+      });
+    }
+
+    // Handle Tables from the source content
+    const tables = main.querySelectorAll('table');
+    if (tables.length > 0) {
+      tables.forEach((table) => {
+        const cells = [
+          ['Table'],
+          [table.outerHTML],
+        ];
+        const newTable = WebImporter.DOMUtils.createTable(cells, document);
+        table.parentElement.append(newTable);
+        table.remove();
+      });
+    }
+
+    // Handle bold text
+    const boldText = main.querySelectorAll('span.cmp-text__body-small-regular, span.cmp-text__body-normal-medium');
+    if (boldText.length > 0) {
+      boldText.forEach((txt) => {
+        const newText = document.createElement('STRONG');
+        newText.innerHTML += txt.innerHTML;
+        txt.replaceWith(newText);
+      });
+    }
+
+    // Need this to fix issue where b>span does not translate well like span>b
+    const boldSpans = main.querySelectorAll('b span');
+    if (boldSpans.length > 0) {
+      boldSpans.forEach((boldSpan) => {
+        boldSpan.innerHTML = `<b>${boldSpan.textContent.trim()}</b> `;
+        if (boldSpan.parentElement) boldSpan.parentElement.replaceWith(boldSpan);
+      });
+    }
+
+    // Fix empty strong tags
+    const strongs = main.querySelectorAll('strong');
+    if (strongs.length > 0) {
+      strongs.forEach((s) => {
+        if (!s.textContent) s.remove();
+      });
+    }
+
+    // Check if the page has a Document Information right nav and import it as a block instead
+    const documentInfoSpan = main.querySelector('span.cmp-text__eyebrow-eyebrow');
+    if (documentInfoSpan && documentInfoSpan.innerHTML === 'Document Information') {
+      // Update the text to H5
+      const newDocInfoSpan = document.createElement('H5');
+      newDocInfoSpan.innerHTML += documentInfoSpan.innerHTML.trim();
+      documentInfoSpan.parentElement.prepend(newDocInfoSpan);
+      documentInfoSpan.remove();
+      const docInfo = newDocInfoSpan.parentElement.parentElement;
+      if (docInfo) {
+        // Add Right Nav block
+        const cells = [
+          ['Right Nav'],
+          [docInfo.innerHTML],
+        ];
+        const table = WebImporter.DOMUtils.createTable(cells, document);
+        main.append(table);
+        // remove elements already added to blocks from main
+        docInfo.remove();
+      }
+    }
+
```

Comment:
Can we add a check to the import script to map additional fields like the Category, Parent, and Product Family to the metadata section, especially for Curam/SPM support pages?

For instance, if the page is `/support/spm/*` or `/support/curam/*`, map the following metadata field to the sharepoint doc structure.

`Category` - `Health and human services`
`Parent` - `Social Program Management`
`Product Family` - `Social Program Management Top `

@keith-kaplan Let me know if there are other metadata fields that you'd like to add as well.

Reply:
Yes we can do this in the import code - BUT we can also do this via bulk metadata especially since it is limited by path. 
Putting it in the individual page metadata would imply that authors should be changing this (which I don't think is the use-case here since this is all analytics related IIRC)?